apps/mobile/src/components/FilterOffersScreen/components/LocationSearch/index.tsx

```diff
@@ -94,19 +95,38 @@ function LocationSearch({
     }
   }, [debouncedSearchValue, searchLocation])
 
+  useEffect(() => {
+    const timeout = setTimeout(() => {
```

Comment:
Should this be here? What if component is mounted but the `visible` is false? Wouldn't this just focus the input even if it is not visible?

Reply:
sry, my bad, forgot to delete it after I moved it to Modals onShow() func